wp-content/themes/missilethreat/assets/_scss/components/_table-of-contents.scss

```diff
@@ -26,6 +31,14 @@
       padding-bottom: rem(24px);
```

Comment:
Instead of `padding`, I would change this to margin. Otherwise, you can actually trigger a hover effect when your cursor is below the actual "link" text.